gestalt/vault.py

```diff
@@ -115,9 +160,12 @@ def get(
             response = self.vault_client.read(path)
```

Comment:
I think you can just implement the retry like

```python
response = retry_call(f=self.vault_client.read,
                                     fargs=[path],
                                     tries=self.tries,
                                    delay=self.delay)
```

and thats basically all you should have to do to implement vault request retries in this entire PR.

Reply:
there are actually a few more spots. Two in the `worker` method (not sure why github isn't letting me make comments on those lines). We also do auth in the `init` method (we shouldn't though, maybe worth changing that in a subsequent PR).

I still think we should use `retry_call` only when utilizing the `self.vault_client` for requests, instead of making additional methods in this class.

Reply:
Makes sense, will make those changes.